README.md

```diff
@@ -37,6 +37,7 @@ This extension allows you to create a code review file you can hand over to a cu
     - [`code-review.priorities`](#code-reviewpriorities)
     - [`code-review.gitDirectory`](#code-reviewgitdirectory)
     - [`code-review.filterCommentsByCommit`](#code-reviewfiltercommentsbycommit)
+    - [`filterCommentsByFilename`](#filtercommentsbyfilename)
```

Comment:
```suggestion
    - [`code-review.filterCommentsByFilename`](#filtercommentsbyfilename)
```

Reply:
Sorry for the typos, and thank you for the review.